addon/components/o-s-s/power-select.hbs

```diff
@@ -0,0 +1,31 @@
+<div class="upf-power-select fx-col" ...attributes {{did-insert (fn this.checkBlockPresence (has-block 'selected-item') 'selected-item')}}
+  {{did-insert (fn this.checkBlockPresence (has-block 'option-item') 'option-item')}}>
```

Comment:
can you invert the params of `checkBlockPresence`, it makes more sense to have the name first, then the value.
Also, what about naming it `ensureBlockPresence` given that it throws an error?